assessment/assessment-7/assessment-7.js

```diff
@@ -0,0 +1,94 @@
+/* ----- Challenge-1 ----- */
+
+function countAE(str1, str2) {
+
+    var str1ToLowerCase = str1.toLowerCase();
+    var str2ToLowerCase = str2.toLowerCase(); 
+
+    var countStr1A = str1ToLowerCase.replaceAll("a", "").length;
+    var countStr1E = str1ToLowerCase.replaceAll("e", "").length;
+    var countStr2A = str2ToLowerCase.replaceAll("a", "").length;
+    var countStr2E = str2ToLowerCase.replaceAll("e", "").length;
+
+    var totalCountStr1 = countStr1A + countStr1E;
+    var totalCountStr2 = countStr2A + countStr2E;
+
+    if (totalCountStr1 === totalCountStr2) {
+        return `${str1}, ${str2}`;
+    } else if (totalCountStr1 > totalCountStr2) {
+        return str1;
+    } else {
+        return str2;
+    }
+}
+
```

Comment:
Lets say we have str1 = "aeaeae"; str2 = "wwwwwwwwwwwwae"; In this case it will fail. We have to count number of a an e. and compare